site/about/deployment/deployment-options.qmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Deployment options"
+date: last-modified
+listing:
+  - id: validmind-deployment-options
+    type: grid
+    max-description-length: 250
+    sort: false
+    grid-columns: 2
+    fields: [title, description]
+    contents:
+      - multi-tenant-cloud.qmd
+      - virtual-private-validmind.qmd
+---
+
+Choose the {{< var vm.product >}} deployment option that best suits your organization, ensuring secure data protection and seamless integration with your existing environment.
+
+## Available options
+
+Our deployment options provide a balance of control and cost-efficiency while integrating seamlessly with your infrastructure. For added flexibility, you can deploy on Amazon Web Services (AWS), Google Cloud Platform (GCP), or Microsoft Azure.
+
+We offer two deployment models:
+
+- **Multi-tenant cloud** — Multiple organizations (tenants) share infrastructure while keeping data isolated, providing cost-efficiency and scalability. VPC support allows secure network connections.
+
+- **Virtual Private ValidMind (VPV)** — A single-tenant architecture where one organization uses a dedicated infrastructure instance, offering greater control, customization, and enhanced security. This option is ideal for high-compliance needs and also supports VPC for secure connectivity.
```

Comment:
Virtual Private ValidMind (VPV) — A single-tenant architecture where one organization uses dedicated infrastructure, offering greater control, customization, and enhanced security. This option is ideal for high-compliance needs, and secure connectivity can also be established via a private link.

Reply:
Changed verbatim to your suggestion.